app/helpers/provisioning_templates_helper.rb

```diff
@@ -1,11 +1,6 @@
 module ProvisioningTemplatesHelper
   def combination(template)
-    template.template_combinations.map do |comb|
-      str = []
-      str << (comb.hostgroup_id.nil? ? _("None") : comb.hostgroup.to_s)
-      str << (comb.environment_id.nil? ? _("None") : comb.environment.to_s)
-      str.join(" / ")
-    end.to_sentence
+    template.template_combinations.map { |comb| comb.hostgroup.to_s }.to_sentence
```

Comment:
Can't hostgroup still be nil?

Reply:
Well logically it should not, having template combination with no association makes no sense :)
Technically I hope not xD